pmd-kotlin/src/main/java/net/sourceforge/pmd/lang/kotlin/KotlinLanguageModule.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.kotlin;
+
+import net.sourceforge.pmd.lang.BaseLanguageModule;
+
+/**
+ * Language Module for Kotlin
+ */
+public class KotlinLanguageModule extends BaseLanguageModule {
```

Comment:
@jsotuyod @adangel Actually, why do we need a PMD Language service for CPD-only languages? They're just dummy implementations but all CPD-only modules have them. They have no use since LanguageRegistry filters them out (with `findWithRuleSupport`), and they could be sources of bugs because the only thing that gives away that they're unsupported is that they return some nulls and throw some UnsupportedOperationExceptions... Should we end this practice?

Reply:
Yes, the PMD Module is not required in that case... and we should not add it then....